ui/app/components/mount-backend/type-form.hbs

```diff
@@ -0,0 +1,37 @@
+{{#each (array "generic" "cloud" "infra") as |category|}}
```

Comment:
Moved the mount type selection to its own component so mount-backend-form doesn't get too overloaded